src/AnalyticsEngine/App.ControlPanel.Engine/SqlExtentions/Profiling-03-CreateSchema.sql

```diff
@@ -556,7 +584,77 @@ IF TYPE_ID(N'ut_yammer_device_activity_log') IS NULL CREATE TYPE ut_yammer_devic
 );
 GO
 
+
+-- Add Copilot TYPE
+
+-- TODO
+
+IF TYPE_ID(N'ut_copilot_activity_log') IS NULL CREATE TYPE ut_copilot_activity_log AS TABLE (
+  [user_id] INT NOT NULL,
+  [date] DATETIME NOT NULL,
+
+  [copilot_chats_count] BIGINT NOT NULL,
+  [copilot_meetings_count] BIGINT NOT NULL,
+  [copilot_files_count] BIGINT NOT NULL
+);
+GO
+
+
+
 -- Add SPs
+
+
+-- ADD COPILOT PRODECURE here
+
+-- TODO
+
+CREATE PROCEDURE [profiling].[usp_UpsertCopilot] (
+  @StartDate DATE, @EndDate DATE
+) AS
+BEGIN
+  SET NOCOUNT ON;
+  DECLARE @copilot AS ut_copilot_activity_log;
+
+  INSERT INTO @copilot (
+    [user_id], [date],
+    [copilot_chats_count],  [copilot_meetings_count], [copilot_files_count]
+
+
+  )
+  SELECT
+    [user_id], @StartDate,
+    SUM([copilot_chats_count]), SUM([copilot_meetings_count]), SUM([copilot_files_count])
+
+  FROM dbo.[copilot_activity_log]
```

Comment:
This table does not exist in the dbo schema.
You could use a new temporary table that you fill in before this SELECT in order to prepare the metrics to be upserted.